aurelia-slickgrid/src/aurelia-slickgrid/index.ts

```diff
@@ -44,6 +44,7 @@ export function configure(aurelia: any, callback: any) {
   aurelia.container.registerTransient(Filters.compoundDate);
   aurelia.container.registerTransient(Filters.compoundInput);
   aurelia.container.registerTransient(Filters.input);
+  aurelia.container.registerTransient(Filters.slider);
```

Comment:
do you need to register the compoundSlider too?

Reply:
good catch, I created the Compound Slider just last night and completely forgot to add it in there, I'll add it

Reply:
done